tests/testCameraMapper.py

```diff
@@ -52,6 +52,9 @@ def __init__(self):
         lsst.obs.base.CameraMapper.__init__(self, policy=policy, repositoryDir=testDir, root=testDir)
         return
 
+    @classmethod
+    def getPackageDir(cls):
+        return "/path/to/nowhere"
```

Comment:
why do these functions need to be implemented in the test class? (why is the base class implementation not sufficient?)